cmd/kpod/images.go

```diff
@@ -55,155 +84,246 @@ func imagesCmd(c *cli.Context) error {
 	if err := validateFlags(c, imagesFlags); err != nil {
 		return err
 	}
-	config, err := getConfig(c)
-	if err != nil {
-		return errors.Wrapf(err, "Could not get config")
-	}
-	store, err := getStore(config)
+
+	runtime, err := getRuntime(c)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "Could not get runtime")
 	}
+	defer runtime.Shutdown(false)
 
-	quiet := false
-	if c.IsSet("quiet") {
-		quiet = c.Bool("quiet")
-	}
-	noheading := false
-	if c.IsSet("noheading") {
-		noheading = c.Bool("noheading")
-	}
-	truncate := true
-	if c.IsSet("no-trunc") {
-		truncate = !c.Bool("no-trunc")
-	}
-	digests := false
-	if c.IsSet("digests") {
-		digests = c.Bool("digests")
-	}
-	outputFormat := genImagesFormat(quiet, truncate, digests)
+	var format string
 	if c.IsSet("format") {
-		outputFormat = c.String("format")
+		format = c.String("format")
+	} else {
+		format = genImagesFormat(c.Bool("quiet"), c.Bool("noheading"), c.Bool("digests"))
+	}
+
+	opts := imagesOptions{
+		quiet:     c.Bool("quiet"),
+		noHeading: c.Bool("noheading"),
+		noTrunc:   c.Bool("no-trunc"),
+		digests:   c.Bool("digests"),
+		format:    format,
 	}
 
-	name := ""
+	imageInput := ""
```

Comment:
I think this can be simplified. Just check `if len(c.Args()) == 1` first and return an error if not, then do `imageInput := c.Args().Get(0)` after - only four lines, no else-if

Reply:
fixed